content/blog/pulumi-service-improvements_02-2020/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Pulumi Console Improvements, February 2020"
+h1: "Pulumi Console Improvements, February 2020"
+authors: ["chris-smith"]
+tags: ["pulumi","new-features"]
```

Comment:
Under the assumption that tags are useful for SEO, could we add more here? For example, pulumi ci, cicd, travis, gitlab, azure pipelines, pulumi audit logs etc.

Reply:
I'd add that we should also use these sparingly and thoughtfully, with an eye toward reuse. They are helpful for SEO, but at the same time, the "tag cloud" grows less useful with each new one that gets added, and when the same post appears on multiple tag pages, it may even reduce the overall SEO impact. IMO, less is more here.

Reply:
Got it. Thanks for the explanation, Christian!